src/components/ReadMore/ReadMore.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useRef, useEffect } from 'react';
+import classnames from 'classnames';
+import { getCommonProps } from '../../utils';
+import Button from '../Button/Button';
+import { ButtonVariants } from '../Button/types';
+import { Collapsible, CollapsibleContent, CollapsibleTrigger } from '../Collapsible';
+import PlusIcon from '../../assets/plus.svg?react';
+import MinusIcon from '../../assets/minus.svg?react';
+
+export interface ReadMoreProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Unique id for component testing
+   */
+  id?: string;
+  /**
+   * Child element pass in to display as item content.
+   */
+  children?: React.ReactNode;
+  /**
+   * Text for the "Read More" button
+   */
+  readMoreText?: string;
+  /**
+   * Text for the "Read Less" button
+   */
+  readLessText?: string;
+  /**
+   * Maximum height of the content when collapsed
+   */
+  maxHeight?: number;
+}
+
+/**
+ * ## Overview
+ *
+ * A component for displaying expandable content with a "Read More" functionality.
+ *
+ * This component is a wrapper around the Collapsible and Button component and uses the CollapsibleTrigger and CollapsibleContent components to control the expand and collapse functionality.
+ *
+ * [Figma Link](https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=7755-5572&t=JCYbkM8yQcdb51UQ-4)
+ *
+ */
+const ReadMore = ({
```

Comment:
Would you mind adding `forwardRef` to this, the component template has been updated now to use that if you look at the latest in main